AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -315,7 +328,8 @@ export class AddParticipantComponent extends BookingBaseComponent implements OnI
             displayName: this.participantDetails.display_name || '',
             companyName: this.participantDetails.company ? this.participantDetails.company : '',
             companyNameIndividual: this.participantDetails.company ? this.participantDetails.company : '',
-            representing: this.participantDetails.representee ? this.participantDetails.representee : ''
+            representing: this.participantDetails.representee ? this.participantDetails.representee : '',
```

Comment:
I think you can just use `||` here, instead of the ternary

Reply:
update to ||